environment/src/helpers/resources.rs

```diff
@@ -103,15 +103,15 @@ impl Resources {
             while let Some(request) = receiver.recv().await {
                 match request {
                     ResourceRequest::Register(resource, id) => {
-                        let resource_name = match resource {
+                        let _resource_name = match resource {
                             Resource::Task(..) => "task",
                             Resource::Thread(..) => "thread",
                         };
 
                         if resources.insert(id, resource).is_some() {
                             error!("A resource with id {} already exists!", id);
                         } else {
-                            trace!("Registered a {} as resource {}", resource_name, id);
+                            // trace!("Registered a {} as resource {}", _resource_name, id);
```

Comment:
If these aren't necessary can we remove the commented out traces?

Reply:
Sure thing.